cl/singlenode/singlenode.go

```diff
@@ -270,7 +270,9 @@ func (app *SingleNodeApp) runLoop() {
 
 			if err != nil {
 				if errors.Is(err, blockbuilder.ErrEmptyBlock) {
-					app.logger.Info("empty block produced, waiting for new payload")
+					noPendingTxesTimeout := 10 * time.Millisecond
```

Comment:
This is the polling interval for checking pending tx count again after it was queried to be zero